v5/heroku.go

```diff
@@ -215,23 +214,30 @@ type Account struct {
 	Beta                bool      `json:"beta" url:"beta,key"`                     // whether allowed to utilize beta Heroku features
 	CreatedAt           time.Time `json:"created_at" url:"created_at,key"`         // when account was created
 	DefaultOrganization *struct {
-		ID   string `json:"id" url:"id,key"`     // unique identifier of organization
-		Name string `json:"name" url:"name,key"` // unique name of organization
-	} `json:"default_organization" url:"default_organization,key"` // organization selected by default
+		ID   string `json:"id" url:"id,key"`     // unique identifier of team
+		Name string `json:"name" url:"name,key"` // unique name of team
+	} `json:"default_organization" url:"default_organization,key"` // team selected by default
+	DefaultTeam *struct {
```

Comment:
So many irrelevant changes to the generated code here 😬

Reply:
Yeah...I wasn't sure why.